apps/web/pages/settings/my-account/profile.tsx

```diff
@@ -72,32 +72,24 @@ interface DeleteAccountValues {
 
 type FormValues = {
   username: string;
-  avatar: string | null;
```

Comment:
Why did you remove null ? Avatar would be null if we want to remove it

Reply:
Okay you added this in updateProfile 
"input.avatar ? await getAvatarToSet(input.avatar) : null"

Reply:
I removed it because in case it is null
![image](https://github.com/calcom/cal.com/assets/86384652/009c954a-d22c-46cd-8c57-247abc2ec395)
The *previewSrc* prop for OrganizationMemberAvatar component will be null, and it uses Avatar component under the hood.
![image](https://github.com/calcom/cal.com/assets/86384652/f42bfb01-3e2a-469a-8f43-1ed7e0c5a039)

Which if provided null value for previewSrc, will just fetch the current avatar of the user. 

But on the profile settings page we would like to show the default avatar svg. 
![image](https://github.com/calcom/cal.com/assets/86384652/8314136f-5f89-404f-b393-8a1b75dc4025)

So using an empty string for that purpose work as a trick. I could do it to support null but dont want regression at some other place.